src/services/lease-service/adapters/xpand/rental-object-adapter.ts

```diff
@@ -0,0 +1,375 @@
+import { logger } from 'onecore-utilities'
+import Config from '../../../../common/config'
+import knex from 'knex'
+import { RentalObject } from 'onecore-types'
+
+const db = knex({
+  client: 'mssql',
+  connection: Config.xpandDatabase,
+})
```

Comment:
I think you should put this in a shared xpandDb or similar, like its done with the db client to leasing local db :) This way all local leasing db queries use the same client and all the xpandDb queries use the same client too